docsbox/docs/via_controller.py

```diff
@@ -5,10 +5,11 @@
 def get_file_from_via(file_id):
     url=app.config["VIA_URL"] + "/" + file_id
     cert=app.config["VIA_CERT_PATH"]
-    return get(url=url, cert=cert, stream=True)
+    return get(url=url, cert=cert, stream=True, timeout=60)
 
 def save_file_on_via(file_path, mime_type, via_allowed_users):
-    data = open(file_path, "rb")
-    cert=app.config["VIA_CERT_PATH"]
-    headers = {'VIA_ALLOWED_USERS': via_allowed_users, 'Content-type': mime_type}
-    return post(url=app.config["VIA_URL"], cert=cert, data=data, headers=headers)
+    with open(file_path, "rb") as data:
+        cert=app.config["VIA_CERT_PATH"]
```

Comment:
Fix the spacing between the `=` signs while you're at it. There are several places where they're not aligned